packages/client/src/index.ts

```diff
@@ -736,6 +740,22 @@ export async function connectScryptedClient(options: ScryptedClientOptions): Pro
                 queryToken,
                 authorization,
                 cloudAddress,
+            },
+            connectShell() {
```

Comment:
I don't think adding a new method specifically for a shell here is necessary. Maybe what you want is extraHeaders?

Reply:
Actually I think you can append the queryToken stuff that's already there to authorize any url like engine.io shell endpoint.